homeassistant/components/overkiz/sensor.py

```diff
@@ -475,6 +475,23 @@ class OverkizSensorDescription(SensorEntityDescription):
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    OverkizSensorDescription(
+        key=OverkizState.CORE_REGULATION_MODE,
+        icon="mdi:auto-mode",
```

Comment:
let's set the icons of these entities in the icons.json as well

Reply:
Done. Let me know if it's the proper to do it